skorch/net.py

```diff
@@ -1055,8 +1073,27 @@ def _get_params_for_optimizer(self, prefix, named_parameters):
     def _get_param_names(self):
         return self.__dict__.keys()
 
+    def _get_params_callbacks(self, deep=True):
```

Comment:
Which problem does this solve and how? (short documentation)

Reply:
Added a comment